### DISCUSSION TITLE
Add: add channels from dailymotion

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#258] [Q&A] [answered]
this channels from israel work on dailymotion in FULL HD 
can you extract links 
Kan 11
https://www.kan.org.il/live/tv.aspx?stationid=2
Kan 33 
https://www.makan.org.il/live/tv.aspx?stationid=25

**Comment by BellezaEmporium** [comment, marked as answer]
Geolocked, won't work.

**Reply by keystroke3** [reply]
If it is available in UK, it might work with `https:salvahe.co/headless?url=`

**Reply by BellezaEmporium** [reply]
It seems that it came back online.

**Comment by tarrazuapps** [comment]
It is possible to get a valid stream of a daily motion broadcast using the API. But it will only work for an amount of time and only for the IP that originally requested it.
